### ISSUE TITLE
Use the model designed by myself.

### ISSUE THREAD

**Issue by Lucifer-Morning-Star** [dotnet/machinelearning#5208]
Hi

I am new to ML, so this might seem like a stupid question, may be it is wrong, but when I learn the ML.NET, I found that all of the models or the way training data in ML.NET have been designed, or you should import it from tensorflow or OXXN.  However, is it possible use the model that the structure of the neural network is designed by myself which has the pooling layers , convolutional layers and so on, just like Tensorflow and Pytorch to trainging the model? Also, could I change the  loss function which designed by myself?

**Comment by Lynx1820** [comment]
Hi @Lucifer-Morning-Star, 

No. We don't support CNN training within ML.NET at this point. The way to circumvent that is by creating your model/ training in another framework like tensorflow and then consuming the model in ML.NET to use for **inferencing** only. Closing this issue, feel free to reopen if you have additional questions.

**Comment by Lucifer-Morning-Star** [comment]
So, is there a plan that would be support in the future? @Lynx1820 

> Hi @Lucifer-Morning-Star,
> 
> No. We don't support CNN training within ML.NET at this point. The way to circumvent that is by creating your model/ training in another framework like tensorflow and then consuming the model in ML.NET to use for **inferencing** only. Closing this issue, feel free to reopen if you have additional questions.

**Comment by Lynx1820** [comment]
Not at the moment.
